esp/bindings/SOAP/Platform/soapservice.cpp

```diff
@@ -129,6 +129,34 @@ int CSoapService::processHeader(CHeader* header, IEspContext* ctx)
     return returnValue;
 }
 
+void CSoapService::addMessageToTraceSummary(IEspContext* ctx, CSoapValue* soapValue, const char* parent)
```

Comment:
Usually, ESP is not configured to use the LogRequest option which make it difficult to debug.